crates/swc_ecma_transforms_react/Cargo.toml

```diff
@@ -35,3 +35,4 @@ swc_ecma_transforms_compat = {version = "0.78.0", path = "../swc_ecma_transforms
 swc_ecma_transforms_module = {version = "0.89.0", path = "../swc_ecma_transforms_module"}
 swc_ecma_transforms_testing = {version = "0.68.0", path = "../swc_ecma_transforms_testing"}
 testing = {version = "0.18.0", path = "../testing"}
+swc = {version = "0.145.1", path = "../swc"}
```

Comment:
This will block publishing.

Reply:
Please remove this

Reply:
Hey thanks for the quick feedback. 
I can remove that however I believe that you won't be able to run the example file without it, as it will be missing imports from that crate (`Compiler` and `Options`)

Reply:
Compiler is not require to use this crate